.gitmodules

```diff
@@ -47,3 +47,6 @@
 [submodule "src/doc/embedded-book"]
 	path = src/doc/embedded-book
 	url = https://github.com/rust-embedded/book.git
+[submodule "src/libunwind/libunwind"]
```

Comment:
shouldn't this be the libunwind from the LLVM monorepo?

Reply:
Especially since we already have a `llvm-project` submodule with a copy of `libunwind` in tree, that would be better imo

Reply:
I'm fine switching to `llvm-project`, but that version is behind upstream and is missing some important changes to libunwind that [removed the C++ standard library dependency](https://github.com/llvm/llvm-project/commit/7fac51724f8875fb063a5ec7079b0172b3387717). We need to roll it first.